src/poprox_recommender/components/rankers/top_news_placer.py

```diff
@@ -0,0 +1,73 @@
+from uuid import UUID
+
+from lenskit.pipeline import Component
+from pydantic import BaseModel
+
+from poprox_concepts.domain import CandidateSet, ImpressedSection
+
+
+class TopNewsPlacerConfig(BaseModel):
+    max_top_news: int = 3
+    total_slots: int = 12
+    top_news_entity_id: UUID | None = None
+    add_section_metadata: bool = True
+
+
+class TopNewsPlacer(Component):
+    config: TopNewsPlacerConfig
+
+    def __call__(
+        self,
+        ranked_articles: ImpressedSection,
+        top_news_article_ids: set[UUID] | None = None,
+        top_news_candidates: CandidateSet | None = None,
```

Comment:
Do we need to support two different input types for this component? Unnecessary complexity increases risk of bugs (now or in the future).